### DISCUSSION TITLE
Enabling plugins from configuration file?

### DISCUSSION THREAD

**Discussion by jereklas** [oxc-project/oxc#4546] [Q&A] [answered]
How do you enable/disable plugins with a configuration file?

I'm wanting some of the disabled plugins enabled so that I can get IDE feedback from the VSCode extension.

**Comment by DonIsaac** [comment]
#5046

**Comment by Boshen** [comment, marked as answer]
https://oxc.rs/docs/guide/usage/linter/config.html#plugins
